odin/adapters/adapter.py

```diff
@@ -199,7 +199,7 @@ def wrapper(_self, path, request):
             """Inner method wrapper."""
             # Validate the Content-Type header in the request against allowed types
             if 'Content-Type' in request.headers:
-                logging.debug("Request content type: %s", request.headers['Content-Type'])
+                # logging.debug("Request content type: %s", request.headers['Content-Type'])
```

Comment:
Uncomment?

Reply:
Should be removed rather than uncommented, otherwise it will be output for every request.